src/js/components/Button/__tests__/Button-test.js

```diff
@@ -569,4 +569,18 @@ describe('Button', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test(`line-height should be zero for icon only`, () => {
+    const { getByRole } = render(
+      <Grommet
+        theme={{
+          button: { default: {} },
+        }}
```

Comment:
Button should not have default defined in the theme. Default should only be defined if using button kind.

```suggestion
```

Reply:
@jcfilben - Thanks for catching that! Removed the theme.